src/app/components/filter/filter-panel.jsx

```diff
@@ -23,21 +23,34 @@ const useStyles = makeStyles({
     background: '#fafdff',
     display: 'grid',
     gridTemplateColumns: '1fr',
-    padding: '16px',
     rowGap: '16px',
+    maxHeight: '64vh',
     [theme.breakpoints.up('sm')]: {
       gridTemplateColumns: '1fr 1fr',
     },
   },
+  filters:{
+    height: '100%',
+    borderRight: '1px solid #599bde',
+    padding: '16px',
+    zIndex: 15,
+  },
+  subfilters:{
+    zIndex:10,
+    background: 'white',
+    padding: '16px',
+    boxShadow:'inset rgb(149 157 165 / 30%) 10px 0px 20px 0px'
+  },
   filterBold: {
     fontWeight: '600',
   },
   filterContainer: {
     display: 'grid',
-    gridTemplateColumns: 'auto',
+    gridTemplateColumns: '1fr 1fr',
```

Comment:
This looks like an opportunity for the RAM style. Something like

```
repeat(auto-fit, minmax(128px, 1fr))
```

This would let them be side-by-side with a minimum width of 128px on every screen, and stack as the screen gets smaller. We need to be prepared for something like 20 of these filters by the time we get to the main search page

Reply:
Ill be using repeat(auto-fit, minmax(48%, 1fr)) so we get on two on each line. It responsive